package.json

```diff
@@ -12,6 +12,7 @@
     "echarts-for-react": "^2.0.1",
     "env-cmd": "10.1.0",
     "history": "5.0.0",
+    "http-proxy-middleware": "^2.0.4",
```

Comment:
What does `http-proxy-middleware` do in this PR

Reply:
It's convenient to proxy api